apps/gallery/test/marionette/activity_test.js

```diff
@@ -57,13 +57,13 @@ marionette('Gallery Activity Tests', function() {
     client.apps.switchToApp(Gallery.ORIGIN);
 
     // Select image from thumbnail list in gallery app
-    galleryApp.tapFirstThumbnail();
+    fullscreenView = galleryApp.tapFirstThumbnail();
 
```

Comment:
We can refactor it further by creating instance of fullscreenView in setup and accessing it here after fullscreenView displayed check. I think we are doing something similar in SMS marionette tests. This way we are avoiding creating a new fullscreenview instance everytime a user tap first thumbnail.